apps/back-office/src/app/(dashboard)/clients/components/CreateClientDrawer.tsx

```
@@ -341,9 +359,11 @@ export function CreateClientDrawer({
       if (updatedErrors[activeTab]) {
         const tabErrors = { ...updatedErrors[activeTab] };
         // Remove errors for fields that now have values
-        if (firstName) delete tabErrors.legalFirstName;
-        if (lastName) delete tabErrors.legalLastName;
-        if (selectedClientParam.email) delete tabErrors.emails;
+        if (selectedClientParam.legal_first_name)
+          delete tabErrors.legalFirstName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)